plugin/db/mysql/mysql.go

```diff
@@ -583,65 +583,17 @@ func (driver *Driver) SetupMigrationIfNeeded(ctx context.Context) error {
 	return nil
 }
 
-// CheckOutOfOrderVersion will return versions that are higher than the given version.
-func (Driver) CheckOutOfOrderVersion(ctx context.Context, tx *sql.Tx, namespace string, source db.MigrationSource, version string) (minVersionIfValid *string, err error) {
-	const checkOutOfOrderVersionQuery = `
-		SELECT MIN(version) FROM bytebase.migration_history
-		WHERE namespace = ? AND source = ? AND version > ?
-		`
-	row, err := tx.QueryContext(ctx, checkOutOfOrderVersionQuery,
-		namespace, source.String(), version,
-	)
-	if err != nil {
-		return nil, util.FormatErrorWithQuery(err, checkOutOfOrderVersionQuery)
-	}
-	defer row.Close()
-
-	var minVersion sql.NullString
-	row.Next()
-	if err := row.Scan(&minVersion); err != nil {
-		return nil, err
-	}
-
-	if minVersion.Valid {
-		return &minVersion.String, nil
-	}
-
-	return nil, nil
-}
-
-// FindBaseline retruns true if any baseline is found.
-func (Driver) FindBaseline(ctx context.Context, tx *sql.Tx, namespace string) (hasBaseline bool, err error) {
-	const findBaselineQuery = `
-		SELECT 1 FROM bytebase.migration_history
-		WHERE namespace = ? AND type = 'BASELINE'
-		`
-	row, err := tx.QueryContext(ctx, findBaselineQuery,
-		namespace,
-	)
-	if err != nil {
-		return false, util.FormatErrorWithQuery(err, findBaselineQuery)
-	}
-	defer row.Close()
-
-	if !row.Next() {
-		return false, nil
-	}
-
-	return true, nil
-}
-
-// FindNextSequence will return the highest sequence number plus one.
-func (Driver) FindNextSequence(ctx context.Context, tx *sql.Tx, namespace string, requireBaseline bool) (int, error) {
-	const findNextSequenceQuery = `
-		SELECT MAX(sequence) + 1 FROM bytebase.migration_history
+// FindLatestSequence will return the latest sequence number.
+func (Driver) FindLatestSequence(ctx context.Context, tx *sql.Tx, namespace string) (int, error) {
```

Comment:
FindLargestSequence ?

Reply:
Maybe prefer latest over largest for being consistent with wording latest migration history record in other places.

Reply:
Done. As a second thought, word "largest" might fit better.